app/services/search/page_searcher.rb

```diff
@@ -1,7 +1,7 @@
 class Search::PageSearcher
 
   def initialize(params)
-    @queries = params[:search]
```

Comment:
curious about this change - why doesn't it need to grab the `:search` subhash any more? did you change it in the form submission?